py-aiobotocore: New package

a-saitoh-fj opened the pull request:
py-aiobotocore: New package
You need py-botocore(#22257) and py-aioitertools(#22256) to build this OSS.

a-saitoh-fj committed:
py-aiobotocore: New package

adamjstewart commented on var/spack/repos/builtin/packages/py-aiobotocore/package.py:
```diff
@@ -0,0 +1,21 @@
+    pypi = "aiobotocore/aiobotocore-1.2.1.tar.gz"
+
+    version('1.2.1', sha256='58cc422e65fc89f7cb78eca740d241ac8e15f39f6b308cc23152711e8a987d45')
+
```
Can you add:
```python
depends_on('python@3.6:', type=('build', 'run'))
```

a-saitoh-fj committed:
py-aiobotocore: Added python dependencies, and Removed unnecessary whitespace

a-saitoh-fj commented:
Fixed the pointed out point.
- Added python dependencies

And,
I'm sorry, I didn't have enough confirmation.
I deleted it because there was extra space.

a-saitoh-fj commented:
All checks seem to be fine.
If your review is okay, I hope it will be merged.
